MAINTAINERS

```diff
@@ -6,4 +6,5 @@
 # COMMITTERS
 # GitHub ID, Name, Email address
 changweige, Changwei Ge, changweige@gmail.com
-eryugey, Eryu Guan, eguan@linux.alibaba.com
\ No newline at end of file
+eryugey, Eryu Guan, eguan@linux.alibaba.com
+imeoer,Yan Song,imeoer@linux.alibaba.com
```

Comment:
The description of the PR mentions "reviewer", but this change looks to add to the _committers_ section. Looks like a new section needs to be added, similar to https://github.com/containerd/project/blob/main/MAINTAINERS#L23

as you're  updating this file, perhaps you could also fix the missing newline 😅☺️

Reply:
Thanks for pointing it out. Yes, I should add a new section.

Reply:
fixed